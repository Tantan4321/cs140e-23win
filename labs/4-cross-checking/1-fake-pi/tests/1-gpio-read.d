1-gpio-read.o: 1-gpio-read.c ../rpi.h
