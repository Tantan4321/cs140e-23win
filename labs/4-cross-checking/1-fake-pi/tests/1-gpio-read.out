TRACE:0: calling pi code
TRACE:1: GET32(0x20200034) = 0x625558ec
TRACE: pi exited cleanly: 8 calls to random
